int32 max_degree
int32 min_degree
float64[] coefficients
